service/src/main/kotlin/fi/espoo/evaka/shared/controllers/ScheduledOperationController.kt

```diff
@@ -4,52 +4,20 @@
 
 package fi.espoo.evaka.shared.controllers
 
-import fi.espoo.evaka.Audit
-import fi.espoo.evaka.application.PendingDecisionEmailService
-import fi.espoo.evaka.application.cancelOutdatedTransferApplications
-import fi.espoo.evaka.application.removeOldDrafts
-import fi.espoo.evaka.attachment.AttachmentsController
-import fi.espoo.evaka.decision.clearDecisionDrafts
-import fi.espoo.evaka.dvv.DvvModificationsBatchRefreshService
-import fi.espoo.evaka.invoicing.controller.parseUUID
-import fi.espoo.evaka.invoicing.service.VoucherValueDecisionService
-import fi.espoo.evaka.koski.KoskiSearchParams
-import fi.espoo.evaka.messaging.daycarydailynote.deleteExpiredDaycareDailyNotes
-import fi.espoo.evaka.placement.deletePlacementPlans
-import fi.espoo.evaka.reports.freezeVoucherValueReportRows
-import fi.espoo.evaka.shared.async.AsyncJobRunner
-import fi.espoo.evaka.shared.async.ScheduleKoskiUploads
-import fi.espoo.evaka.shared.async.removeOldAsyncJobs
 import fi.espoo.evaka.shared.db.Database
-import fi.espoo.evaka.shared.domain.HelsinkiDateTime
-import fi.espoo.evaka.shared.utils.europeHelsinki
-import fi.espoo.evaka.varda.VardaUpdateService
-import mu.KotlinLogging
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
-import java.time.Instant
-import java.time.LocalDate
-
-private val logger = KotlinLogging.logger { }
 
 @RestController
 @RequestMapping("/scheduled")
-class ScheduledOperationController(
-    private val vardaUpdateService: VardaUpdateService,
-    private val dvvModificationsBatchRefreshService: DvvModificationsBatchRefreshService,
-    private val attachmentsController: AttachmentsController,
-    private val pendingDecisionEmailService: PendingDecisionEmailService,
-    private val voucherValueDecisionService: VoucherValueDecisionService,
-    private val asyncJobRunner: AsyncJobRunner
-) {
+class ScheduledOperationController {
 
     @PostMapping("/dvv/update")
-    fun dvvUpdate(db: Database.Connection): ResponseEntity<Int> {
-        Audit.VtjBatchSchedule.log()
-        return ResponseEntity.ok(dvvModificationsBatchRefreshService.scheduleBatch(db))
+    fun dvvUpdate(db: Database.Connection) {
+        // executed by DailyJobRunner instead
```

Comment:
Does this respond with "no content" implicitly (comparing this and e.g. `/koski/update`)?

Reply:
What about this one ^

Reply:
A void function returns status 200 with empty body (`Content-Length: 0`), so this behaves differently than earlier but it shouldn't matter.

But I think I'll add one more commit and remove the REST controller completely in this pull request...it means the nightly lambdas will fail, but it's just a temporary problem until they're removed as well